reference/CloudPay-API-Specification.yaml

```diff
@@ -1375,12 +1375,177 @@ paths:
         '500':
           description: Error Response
       description: |
-        ![In Review](https://img.shields.io/badge/Status-In%20Review-green?style=flat-square)
+        ![In Development](https://img.shields.io/badge/Status-In%20Development-green?style=flat-square)
 
         The Account Endpoint returns an account summary of the Authenticated User.
 
         **Note** *Only one of the four authorisation parameters are required*
     parameters: []
+    put:
+      summary: Update the User's Summary
+      operationId: put-api-v1-account
+      parameters:
+        - $ref: '#/components/parameters/apiJwtTokenParam'
+        - $ref: '#/components/parameters/apiSessionIdParam'
+        - $ref: '#/components/parameters/thirdPartyAuthCookie'
+        - $ref: '#/components/parameters/authCookie'
+      tags:
+        - Account
+      responses:
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                description: ''
+                type: object
+                properties:
+                  EmailAddress:
```

Comment:
Mandatory fields include password but maybe update password should be in different endpoint?